tmd/Neuron/Neuron.py

```diff
@@ -17,6 +18,34 @@ class Neuron(object):
     from tmd.Neuron.methods import size
     from tmd.Neuron.methods import get_bounding_box
 
+    @classmethod
+    def from_morphio(cls, path_or_obj):
+        """Create Neuron object from morphio object or from path
+        loaded via morphio.
+
+        Args:
+            path_or_obj (Union[str, morphio.Morphology]):
+                Filepath or morphio object
+
+        Returns:
+            neuron (Neuron): tmd Neuron object
+        """
+        from morphio import Morphology
+        from tmd.Neuron.conversion import convert_morphio_soma
+        from tmd.Neuron.conversion import convert_morphio_trees
+
+        if isinstance(path_or_obj, (str, Path)):
+            obj = Morphology(path_or_obj)
+        else:
+            obj = path_or_obj
+
+        neuron = cls()
+        neuron.set_soma(convert_morphio_soma(obj.soma))
+        for tree, types in convert_morphio_trees(obj):
+            neuron.append_tree(tree, types)
```

Comment:
you can import the tree_types dictionary here and use the same one because you should not have a different one for each tree.

Reply:
done